package.json

```diff
@@ -5,8 +5,8 @@
   "scripts": {
     "start": "webpack-dev-server",
     "build": "cross-env NODE_ENV=production webpack",
-    "lint": "eslint src/**/*.{tsx,ts}",
-    "test": "jest",
+    "lint": "eslint 'src/**/*.{ts,tsx}' --fix && prettier 'src/**/*{ts,tsx}' --write",
+    "test": "cross-env TZ=America/New_York jest",
```

Comment:
Needed to set the timezone, as otherwise where you are in the world will give different dates with the ISO string you provide in one of the tests. This could be set to anything, just needs to be set to ensure the snapshots pass.